core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroup.java

```diff
@@ -61,6 +61,23 @@
 /**
  * An {@link EndpointGroup} that filters out unhealthy {@link Endpoint}s from an existing {@link EndpointGroup},
  * by sending periodic health check requests.
+ *
+ * <pre>{@code
+ * EndpointGroup originalGroup = ...
+ *
+ * // Decorate the EndpointGroup with HealthCheckedEndpointGroup
+ * // that sends HTTP health check requests to '/internal/l7check' every 10 seconds.
+ * HealthCheckedEndpointGroup healthCheckedGroup =
+ *         HealthCheckedEndpointGroup.builder(originalGroup, "/internal/l7check")
+ *                                   .protocol(SessionProtocol.HTTP)
+ *                                   .retryInterval(Duration.ofSeconds(10))
+ *                                   .build();
+ *
+ * // You must specify healthCheckedGroup when building a WebClient, otherwise the client would send the
```

Comment:
`, otherwise health checking will not be enabled.`

We can't predict the future of whether there are unhealthy endpoints or not so should just state the facts.

Reply:
That's a good point. Fixed. 😄